### PR TITLE
sig node ci working group

### PR DESCRIPTION
**Pull request by SergeyKanzhelev**
/assign @mrbobbytables 
/sig node

### CODE REVIEW

**Review by pacoxu — APPROVED** [review]
/lgtm

File: groups/sig-node/groups.yaml [hunk 1]
```diff
@@ -54,6 +54,26 @@ groups:
       MembersCanPostAsTheGroup: "false"
       ReconcileMembers: "false"
 
+  - email-id: sig-node-ci@kubernetes.io
+    name: sig-node-ci
+    description: |-
+      SIG CI Working Group discussion group
+    owners:
+      - killen.bob@gmail.com # temporary to finish configuration
+      - sig-node-leads@kubernetes.io # SIG Node as a default owner
```
Comment (mrbobbytables) [inline groups/sig-node/groups.yaml:63]: A group cannot be an owner of another group
ref: https://support.google.com/a/answer/167094

Comment (pacoxu) [reply groups/sig-node/groups.yaml:63]: /lgtm cancel

Comment (SergeyKanzhelev) [reply groups/sig-node/groups.yaml:63]: I thought this is the group: https://github.com/kubernetes/k8s.io/blob/0935943ff2b6c79bac7765a04d46584fa0341499/groups/groups.yaml#L19

It's where I took inspiration from

Comment (SergeyKanzhelev) [reply groups/sig-node/groups.yaml]: fixed

Comment (mrbobbytables) [reply groups/sig-node/groups.yaml]: that one is a named account used by contribex for managing things

Comment (SergeyKanzhelev) [reply groups/sig-node/groups.yaml]: yes, I was just confused by it - the name of is sounds like a group

**Comment by SergeyKanzhelev** [comment]
@mrbobbytables or @pacoxu can any of you re-approve?

**Comment by SergeyKanzhelev** [comment]
/assign @nikhita

as per bot recommendation

**Review by MadhavJivrajani — APPROVED** [review]
/approve
/lgtm

**Comment by MadhavJivrajani** [comment]
in case you want to address the comment ^

File: groups/sig-node/groups.yaml [hunk 2]
```diff
@@ -53,6 +53,30 @@ groups:
...
+    description: |-
+      SIG CI Working Group discussion group
+    owners:
+      # SIG Node as a default owner
+      - dawnchen@google.com
+      - decarr@redhat.com
+      - mpatel@redhat.com
+      - s.kanzhelev@gmail.com
+      - pehunt@redhat.com
+      # Add SIG Node CI group chairs explicitly when they are not in pervious list
```
[the remark this answers is not in the document]
Comment (SergeyKanzhelev) [reply groups/sig-node/groups.yaml]: thank you!

**Comment by SergeyKanzhelev** [comment]
/unhold

fixed spelling. Thank you!

**Pull request merged**